### ISSUE TITLE
Should SOURCE_DOCUMENTS folder only contain NEW documents?

### ISSUE THREAD

**Issue by neural-oracle** [PromtEngineer/localGPT#309]
should the SOURCE_DOCUMENTS folder only contain NEW documents that were not already processed via ingest.py?
So once I run ingest.py on a document successfully I can then remove them from SOURCE_DOCUMENTS since its contents is "memorized" in our chroma-embeddings ?
I hope my understanding is correct... thx

I believe, ingest.py will process all SOURCE_DOCUMENT files even if they were already processed in earlier runs of ingest.py.
I Just want to make sure that I got this right.

**Comment by bp020108** [comment]
No, i have tried this and when you remove docs and when you run ingest.py for new docs then DB folder files are created newly which is not answering for old docs,  Not sure what is the way to solve this? if anyone can help here
